finalPrep/modZeroHero.js

```diff
@@ -2,52 +2,125 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Kitty the Amazing";
+var specialAbility = "a kitty that flies through time and space to fight evil rats";
+
+console.log(heroName);
+console.log(specialAbility);
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = ( "Never fear" + " " + heroName + " " + "is here to save your tails!" );
```

Comment:
the spaces can be included in your stings, just to simplify this a bit. For example "Never fear " will keep that last space character.